apps/taiga-lumbermill/src/dashboards/crypto/components/prices/prices.component.html

```diff
@@ -0,0 +1,21 @@
+<div
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+    [style.height]="'100%'"
+>
+    <header tuiHeader>
+        <h2
+            tuiTitle
+            [style.text-align]="'center'"
+        >
+            Cryptocurrency Prices
+        </h2>
+    </header>
+    @if (chosen !== '') {
+        <lmb-price-chart [chosen]="chosen" />
+    }
+    <lmb-price-list
+        [chosen]="chosen"
+        (chosenChange)="changeValue($event)"
```

Comment:
you can use banana-box syntax instead
`[(chosen)]="chosen"` 
(but let's rename it to token `[(token)]="token"`